### PR TITLE
Change Nidhoggs method of using super_buff

### PR DESCRIPTION
**Pull request by MarianArlt**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

This PR is tightly tied to https://github.com/project-topaz/topaz/pull/1388 and came to be because that PR was found to be the legitimate use case for skill ID 956
(KS99 Wyrm airborne Hurricane Wing).

In the discussion of that PR it was decided to remove the use of any non-accurate IDs for now and instead encourage the re-factoring of the random defense/resistance buff that Nidhogg can use at will.

I commented out rather than deleting anything for later reference.

### CODE REVIEW

**Comment by Era-Lusiphur** [comment]
I haven't looked into it really but was it used that way to make it true AoE, hitting everyone nearby?

**Comment by ibm2431** [comment]
Oh, Nidhogg was already set up in a local var 2hr format. That makes this a lot easier than I thought.

If proper animation / skill can't be found, then we can leave all of onMobFight as is (active), but replace the `useMobAbility` call with a direct invocation of `mob:addStatusEffectEx(tpz.effect.SUPER_BUFF, 0, 50, 0, 30)`. Then [the "super buff" skill can be deleted altogether](https://github.com/project-topaz/topaz/blob/release/scripts/globals/mobskills/super_buff.lua).

If we don't have a capture, we can go without the animation if it means fixing this wonky skill problem. 

I'd be willing to bet that Nidhogg uses VULTURE2 on retail. 😉

**Comment by MarianArlt** [comment]
I searched tons of functions today to see if there's a method to simply play the animationID 432 without having to call a skill but couldn't find any. Played around with inject functions (all player or NPC targeted) and did some extremely funny things with `independantAnimation()` but none of these can do this. Is there no such function?
(Would be perfectly appropriate here)

**Comment by ibm2431** [comment]
From @59blargedy's capture, which I won't link here on GitHub. It's on the Captures Discord. You can see the effect go off at 2:36 in her video. (Also note the animation ID.)
super_buff

ActionView didn't display anything because its check [against Windower's resource table failed](https://github.com/Windower/Resources/blob/master/resources_data/monster_abilities.lua#L794-L795).

File: scripts/globals/mobskills/super_buff.lua [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---------------------------------------------
+-- Spike Flail
+-- Deals extreme damage in a threefold attack to targets behind the user.
```
Comment (ibm2431) [inline scripts/globals/mobskills/super_buff.lua]: Be sure to change the copy-paste comment block 😉

File: scripts/globals/mobskills/super_buff.lua [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---------------------------------------------
+-- Spike Flail
```
Comment (TeoTwawki) [inline scripts/globals/mobskills/super_buff.lua]: fix the comment for us if you would :+1:

**Comment by MarianArlt** [comment]
Awesome. Thank you ibm I have tried to spot that in the videos, looking at that panel but couldn't see it. I appreciate the support.
PR now simply adds the appropriate ID and changes the ID in the script.

File: scripts/globals/mobskills/super_buff.lua [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+-- Deals extreme damage in a threefold attack to targets behind the user.
+---------------------------------------------
+require("scripts/globals/monstertpmoves")
+require("scripts/globals/settings")
```
Comment (ibm2431) [inline scripts/globals/mobskills/super_buff.lua]: I'm not sure why this was in the original Super Buff, but settings aren't used in this one!

File: scripts/globals/mobskills/super_buff.lua (hunk 2 above)
Comment (MarianArlt) [reply scripts/globals/mobskills/super_buff.lua]: Will do.

File: scripts/globals/mobskills/super_buff.lua (hunk 1 above)
Comment (ibm2431) [reply scripts/globals/mobskills/super_buff.lua]: Teo beat me to my comment again. ☹️

Comment (TeoTwawki) [reply scripts/globals/mobskills/super_buff.lua]: jinx

**Review by ibm2431 — APPROVED** [review]

**Review by TeoTwawki — APPROVED** [review]

**Pull request merged**